src/api/model.js

```diff
@@ -34,4 +34,16 @@ app.post('/', (req, res) => {
         });
 });
 
+app.delete('/', (req, res) => {
+    addAction('DELETE_MODEL', {
+        modelAddress: req.query.modelAddress
+    }).then(() => {
+        res.successJson({
+            models: ModelStore.getModels()
```

Comment:
I agree it is atypical. I did this to avoid having two calls go to the server. One would be to delete the item then I would need the newly updated list of models in order to reload the page correctly. If people think that two calls are fine then I will need more time to implement the change.